crates/pop-cli/src/commands/up/mod.rs

```diff
@@ -71,7 +71,14 @@ impl Command {
 			cmd.execute().await?;
 			return Ok("contract");
 		}
-		cli.warning("No contract detected. Ensure you are in a valid project directory.")?;
+		#[cfg(feature = "parachain")]
+		if pop_parachains::is_supported(project_path.as_deref())? {
+			cli.warning("Parachain deployment is currently not implemented.")?;
+			return Ok("parachain");
```

Comment:
```suggestion
			return Ok(Project::Rollup);
```

Why are we using strings when this is exactly what an enum is for? An enum gives type safety and is more efficient. If you need a string version, get it from the enum variant.

Reply:
This feels out of scope here since the entire codebase follows this approach. However, I agree with this change, and introducing this struct will help consolidate all telemetry options in one place. I've created a task and will handle it separately.